packages/modal/components/DialogModal.tsx

```diff
@@ -38,8 +51,28 @@ class DialogModal extends React.PureComponent<DialogModalProps, {}> {
           className={cx(modalHeader, flexItem("shrink"))}
           data-cy="dialogModal-header"
         >
-          <div className={cx(flex({ align: "center" }), padding("all", "l"))}>
-            <div className={cx(flexItem("grow"), textSize("l"))}>{title}</div>
+          <div
+            className={cx(
+              flex({ align: "center", justify: "center" }),
+              padding("all", "l")
+            )}
+          >
+            <div className={cx(flexItem("grow"), textSize("l"))}>
+              {iconShape ? (
+                <Flex gutterSize="xxs" justify="center" align="center">
+                  <FlexItem flex="shrink">
+                    <IconPropAdapter
+                      icon={iconShape}
+                      size={iconSize}
+                      color={iconColor}
+                    />
+                  </FlexItem>
+                  <FlexItem flex="grow">{title}</FlexItem>
+                </Flex>
+              ) : (
+                <>{title}</>
```

Comment:
does this not work without the `<>` ?

Reply:
Yeah, TypeScript 😓 we have a lot of this going on in Kommander too.